src/main/io/util/usb.c

```diff
@@ -99,20 +99,6 @@ void *io_usb_open(uint16_t vid, uint16_t pid, uint16_t config, uint16_t iface)
     return NULL;
   }
 
-  ret = libusb_set_configuration(dev, config);
```

Comment:
That change is also not too obvious. Would be good to have that in a separate commit as well. This might still have a negative impact on anything that is untested, so better visibility is really good to have imo.